.docker/php.ini

```diff
@@ -0,0 +1 @@
+opcache.validate_timestamps = 1
```

Comment:
Unrelated to this PR, but only just spotted.

Default is `0` (https://github.com/elifesciences/elife-base-images/blob/aee1e4b0a25151e508a88a223a3ccf3e99d55c04/config/php-7.0-elife.ini#L13), which is correct for production but useless for development (as changes to PHP files aren't recognised).

Reply:
Should the base image should be optimised for production then overridden for development, or optimised for dev then overridden for production?

Reply:
I tend to go for production-oriented as the base because there are more environments that look similar to that (`end2end`, `continuumtest`, `demo`, ad-hoc ones) while `dev` is only replicated in the `ci` stages

Reply:
👍 We will need to replicate this in other projects (eg Annotations).